include/capstone/m68k.h

```diff
@@ -150,17 +150,20 @@ typedef struct m68k_op_br_disp {
 	uint8_t disp_size;		///< Size from m68k_op_br_disp_size type above
 } m68k_op_br_disp;
 
+/// Register pair in one operand.
+typedef struct cs_m68k_op_reg_pair {
+  m68k_reg reg_0;
```

Comment:
please use tabs for indentation, not spaces.

Reply:
I've converted into tabs now.